sqlx-core/src/any/options.rs

```diff
@@ -19,6 +19,7 @@ use url::Url;
 pub struct AnyConnectOptions {
     pub database_url: Url,
     pub log_settings: LogSettings,
+    pub enable_config: bool,
```

Comment:
Given that these are part of the public interface, they should probably all have doc comments.

Reply:
Probably, but not having a doc for enable_config is consistent with the surrounding code. Documenting all of the options would be a subject for a different pull request.

Reply:
Personally, I'd say that documenting new code is more important than being consistent with undocumented items. enable_config is effectively meaningless as a field name without additional context. Obviously we have it right now, but the docs are for people that come later.